File: base/src/main/java/me/shetj/base/http/callback/EasyCallBack.java
package me.shetj.base.http.callback;

import android.support.annotation.Keep;

import com.zhouyou.http.callback.CallBack;
import com.zhouyou.http.exception.ApiException;

@Keep
public class EasyCallBack<Object> extends CallBack<Object> {
	@Override
	public void onStart() {

	}

	@Override
	public void onCompleted() {

	}

	@Override
	public void onError(ApiException e) {

	}

	@Override
	public void onSuccess(Object o) {

	}
}

File: base/src/main/java/me/shetj/base/http/callback/EasyCallBack.kt
package me.shetj.base.http.callback

import androidx.annotation.Keep

import com.zhouyou.http.callback.CallBack
import com.zhouyou.http.exception.ApiException

@Keep
class EasyCallBack<T> : CallBack<T>() {
    override fun onStart() {

    }

    override fun onCompleted() {

    }

    override fun onError(e: ApiException) {

    }

    override fun onSuccess(o: T) {

    }
}
